aspnetcore/test/troubleshoot-azure-iis.md

```diff
@@ -228,47 +229,43 @@ The [ASP.NET Core Module](xref:host-and-deploy/aspnet-core-module) is configured
 
 ### Azure App Services Log stream
 
-The Azure App Services Log streams logging information as it occurs. To view streaming logs:
-
-1. In the Azure portal, open the app in **App Services**.
-1. In the left pane, navigate to **Monitoring** > **App Service Logs**.
-  ![App Service Logs](https://user-images.githubusercontent.com/3605364/183573538-80645002-d1c3-4451-9a2f-91ef4de4e248.png)
-1. Select **File System** for **Web Server Logging**. Optionally enable **Application logging**.
-  ![enable logging](https://user-images.githubusercontent.com/3605364/183529287-f63d3e1c-ee5b-4ca1-bcb6-a8c29d8b26f5.png)
-1. In the left pane, navigate to **Monitoring** > **Log stream**, and then select **Application logs** or **Web Server Logs**.
-
-  ![Monitoring Log stream](https://user-images.githubusercontent.com/3605364/183561255-91f3d5e1-141b-413b-a403-91e74a770545.png)
-
-  The following images shows the application logs output:
-
-  ![app logs](https://user-images.githubusercontent.com/3605364/183528795-532665c0-ce87-4ed3-8e4d-4b374d469c2a.png)
+The Azure App Services Log streams logging information as it occurs. For more information, see:
 
-Streaming logs have some latency and might not display immediately.
+* [Azure App Service diagnostics overview](/azure/app-service/app-service-diagnostics)
+* [Enable diagnostics logging for web apps in Azure App Service](/azure/app-service/web-sites-enable-diagnostic-log)
+* [Monitor Azure App Service](/azure/app-service/web-sites-monitor)
+* [Azure App Service troubleshooting and diagnostics](/azure/app-service/troubleshoot-diagnostic-logs)
+* [Azure App Service Logging: How to Monitor Your Web Apps in Real-Time](https://techcommunity.microsoft.com/t5/apps-on-azure-blog/azure-app-service-logging-how-to-monitor-your-web-apps-in-real/ba-p/3800390)
+* [Azure App Service log streaming overview](/azure/app-service/web-sites-log-streaming-overview)
+* [Azure App Service log streaming](/azure/app-service/web-sites-log-streaming)
+* [Azure App Service log streaming troubleshooting](/azure/app-service/web-sites-log-streaming-troubleshooting)
+  * [Azure App Service log streaming with Azure CLI](/azure/app-service/web-sites-log-streaming-azure-cli)
+  * [Azure App Service log streaming with Azure PowerShell](/azure/app-service/web-sites-log-streaming-azure-powershell)
+  * [Azure App Service log streaming with Visual Studio](/azure/app-service/web-sites-log-streaming-visual-studio)
+  * [Azure App Service log streaming with Azure Functions](/azure/app-service/web-sites-log-streaming-azure-functions)
+  * [Azure App Service log streaming with Azure DevOps](/azure/app-service/web-sites-log-streaming-azure-devops)
+  * [Azure App Service log streaming with Azure Monitor](/azure/app-service/web-sites-log-streaming-azure-monitor)
```

Comment:
@tdykstra CoPilot put in 10 links to topics that don't exist. LLM's are predictive engines, and those topics should be written, LOL, or maybe they will be written in a couple years.